src/scripts/spot.js

```diff
@@ -204,6 +208,28 @@ determineLimit = function (word) {
     return words[word];
 };
 
+var spotifyAuthOptions = {
+    url:'https://accounts.spotify.com/api/token',
+    form: {'grant_type':'client_credentials'},
+    headers: {'Authorization': 'Basic ' + process.env.HUBOT_SPOTIFY_CREDENTIALS }
+};
+
+function spotifyRequestOptions(callback) {
+    request.post(spotifyAuthOptions, function (err,httpResponse,body) {
+        if (err) {
+            console.log(err, body);
+        } else {
+            var accessToken = JSON.parse(body).access_token;
+            //console.log(accessToken);
```

Comment:
@cmckendry for these commented out `console.log` can you either a) uncomment and update them to use `logger` or b) just remove them? 😄